authors/apps/articles/views.py

```diff
@@ -1,11 +1,12 @@
-from .models import Article
+from .models import Article, Ratings
```

Comment:
Observe the order of imports as we had agreed earlier.

Reply:
oh yeah let me check on that